src/zenml/cli/stack_recipes.py

```diff
@@ -0,0 +1,1016 @@
+        "Do you wish to delete the stack recipes directory? \n"
+        f"{stack_recipes_directory}"
+    ):
+        git_stack_recipes_handler.clean_current_stack_recipes()
+        cli_utils.declare(
+            "Stack recipes directory was deleted from your current working "
+            "directory."
+        )
+    elif not fileio.isdir(stack_recipes_directory):
+        logger.error(
+            f"Unable to delete the stack recipes directory - "
+            f"{stack_recipes_directory} - "
+            "as it was not found in your current working directory."
+        )
+
+
+@stack_recipe.command(help="Find out more about a stack recipe.")
+@pass_git_stack_recipes_handler
+@click.argument("stack_recipe_name")
+def info(
+    git_stack_recipes_handler: GitStackRecipesHandler, stack_recipe_name: str
+) -> None:
+    """Find out more about an stack recipe.
+
+    Outputs a pager view of the stack_recipe's README.md file.
+
+    Args:
+        git_stack_recipes_handler: The GitStackRecipesHandler instance.
+        stack_recipe_name: The name of the stack recipe.
+    """
+    try:
+        stack_recipe_obj = git_stack_recipes_handler.get_stack_recipes(
+            stack_recipe_name
+        )[0]
+    except KeyError as e:
+        cli_utils.error(str(e))
+
+    else:
+        md = Markdown(stack_recipe_obj.readme_content)
+        with console.pager(styles=True):
+            console.print(md)
+
+
+@stack_recipe.command(
+    help="Pull stack recipes straight into your current working directory."
+)
+@pass_git_stack_recipes_handler
+@click.argument("stack_recipe_name", required=False, default=None)
+@click.option(
+    "--yes",
+    "-y",
+    "force",
+    is_flag=True,
+    help="Force the redownload of the stack_recipes folder to the ZenML config "
+    "folder.",
+)
+@click.option(
+    "--path",
+    "-p",
+    type=click.STRING,
+    default="zenml_stack_recipes",
+    help="Relative path at which you want to install the stack recipe(s)",
+)
+def pull(
+    git_stack_recipes_handler: GitStackRecipesHandler,
+    stack_recipe_name: str,
+    force: bool,
+    path: str,
+) -> None:
+    """Pull stack_recipes straight into your current working directory.
+
+    Add the flag --yes or -y to redownload all the stack_recipes afresh.
+    Use the flag --version or -v and the version number to specify
+    which version of ZenML you wish to use for the stack_recipes.
+
+    Args:
+        git_stack_recipes_handler: The GitStackRecipesHandler instance.
+        stack_recipe_name: The name of the stack_recipe.
+        force: Force the redownload of the stack_recipes folder to the ZenML config
+            folder.
+        path: The path at which you want to install the stack_recipe(s).
+    """
+    cli_utils.warning(
+        "The stack recipes CLI is in alpha and actively being developed. "
+        "Please avoid running mission-critical workloads on resources deployed "
+        "through these commands."
+    )
```

Comment:
Maybe this message (and elsewhere where it is used) can be turned into a CONSTANT somewhere to reduce the duplication?